src/gui/optionsdialog.cpp

```diff
@@ -1593,6 +1593,8 @@ int OptionsDialog::getActionOnDblClOnTorrentDl() const
 {
     if (m_ui->actionTorrentDlOnDblClBox->currentIndex() < 1)
         return 0;
+    if (m_ui->actionTorrentDlOnDblClBox->currentIndex() == 2) // this checkbox is missing DoubleClickAction::PREVIEW_FILE
```

Comment:
hmm, maybe it would be better to create a new enum for Downloading Action, changing `DoubleClickAction` would change existing behavior.